test/docfx.Test/lib/JsonSchemaTest.cs

```diff
@@ -52,6 +51,18 @@ public class JsonSchemaTest
             "items and subitems",
             "with boolean schema",
             "patternProperties",
+
+             // ref
+            "relative pointer ref to object",
+            "relative pointer ref to array",
+            "escaped pointer ref",
+            "remote ref, containing refs itself",
+            "$ref to boolean schema true",
+            "$ref to boolean schema false",
+            "Recursive references between schemas",
+            "refs with quote",
+            "root pointer ref/mismatch", // additional properties
```

Comment:
now `additional properties` has been implemented

Reply:
it's already included